src/Loader.php

```diff
@@ -596,7 +595,6 @@ public function clear_cache_twig()
         }
 
         if (is_string($cache_location) && is_dir($cache_location)) {
-            // @todo What if not all files could be deleted?
```

Comment:
Might be improved when we introduce better and streamlined error handling, see https://github.com/timber/timber/issues/2210.